app/components/submission/ManuscriptMetadata/SubjectAreaDropdown.js

```diff
@@ -74,10 +73,106 @@ class SubjectAreaDropdown extends React.Component {
       hasReachedMultiselectLimit: false,
     }
 
+    const chooseBorderColorFromProps = (
+      theme,
+      validationStatus = 'default',
+      isFocused,
+    ) =>
+      ({
+        error: theme.colorError,
+        success: theme.colorSuccess,
+        warning: theme.colorWarning,
+        default: isFocused ? '#2684FF' : theme.colorBorder,
+      }[validationStatus])
+
+    const gridUnitValue = parseInt(this.props.theme.gridUnit, 10)
+
     this.customReactSelectStyles = {
+      valueContainer: (base, state) => ({
+        ...base,
+        padding: `${gridUnitValue / 2}px`, // combines with margin on multiValue to achieve gridUnit spacing around tags
+      }),
+      placeholder: (base, state) => ({
+        ...base,
+        color: this.props.theme.colorText,
+        margin: `${gridUnitValue / 2}px`,
+        padding: `${gridUnitValue / 2}px`,
+      }),
+      input: (base, state) => ({
+        ...base,
+        padding: `${gridUnitValue / 2}px`,
+      }),
       multiValue: (base, state) => ({
         ...base,
         backgroundColor: this.props.theme.colorPrimary,
+        color: this.props.theme.colorTextReverse,
+        margin: `${gridUnitValue / 2}px`, // combines with padding on valueContainer to achieve gridUnit spacing around tags
+      }),
+      multiValueLabel: (base, state) => ({
+        ...base,
+        color: this.props.theme.colorTextReverse,
+        fontSize: this.props.theme.fontSizeBase,
+        padding: this.props.theme.gridUnit,
+        paddingLeft: this.props.theme.gridUnit,
```

Comment:
Isn't this line redundant given the line above?

Reply:
I thought it was needed, to override [the original](https://github.com/JedWatson/react-select/blob/v2/src/components/MultiValue.js#L36) because `paddingLeft` is more specific than `padding`. But maybe not?

Reply:
Oh I see because we're merging JS objects, not actually writing CSS. Carry on.